### ISSUE TITLE
Loss not consistent with mask2former loss

### ISSUE THREAD

**Issue by AniTho** [SooLab/SimCIS#8]
Dear authors
In your paper, the additional loss mentioned is only consistent selection loss but diving deeper into the loss function that you have used with mask2former is different from the ones used in the original repository. In fact, extra loss (loss_bbox_panoptic) is used by you whereas mask2former, eclipse, balconpas and comformer don't use this additional loss term. Can you please provide source of the loss that is used?

**Comment by chenrxi** [comment]
I guess the author's code is based on MaskDino @AniTho . Have you solved your concern?

**Comment by AniTho** [comment]
> I guess the author's code is based on MaskDino [@AniTho](https://github.com/AniTho) . Have you solved your concern?

But isn't it unfair to other papers that the authors compared against? For e.g. the other papers don't use additional loss terms so that raise the question, is the paper performance boost attributed to change in loss and training (from mask dino) or the authors ideas?

**Comment by chenrxi** [comment]
Actually, i think that the author's ideas are largely borrowed from MaskDino and modified into the continual learning setting. the additional loss terms can be seemed as some of the tricks which is also used in CUE(towards continual universal segmentation) and many segmentation algorithms, which is not unfair to some extent.
